src/SFML/Audio/SoundBuffer.cpp

```diff
@@ -80,11 +80,19 @@ sfSoundBuffer* sfSoundBuffer_createFromStream(sfInputStream* stream)
 
 
 ////////////////////////////////////////////////////////////
-sfSoundBuffer* sfSoundBuffer_createFromSamples(const int16_t* samples, uint64_t sampleCount, unsigned int channelCount, unsigned int sampleRate)
+sfSoundBuffer* sfSoundBuffer_createFromSamples(const int16_t*    samples,
```

Comment:
Corresponding declaration was not updated.

Reply:
I'm used to C++ where changing a member function without updating the declaration is a compiler error but alas these are not member functions.